src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -75,51 +81,103 @@ public async Task<(Error, IEnumerable<GitHubUser>)> GetUsersByCommit(string repo
                 return default;
             }
 
-            var apiDetail = $"GET /repos/{repoOwner}/{repoName}/commits/{commitSha}";
+            var queryStr = @"
+query ($owner: String!, $name: String!, $commit: String!) {
+  repository(owner: $owner, name: $name) {
+    object(expression: $commit) {
+      ... on Commit {
+        history(first: 100) {
+          nodes {
+            author {
+              email
+              user {
+                databaseId
+                name
+                login
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}";
+
+            var request = new
+            {
+                query = queryStr,
+                variables = new
+                {
+                    owner = repoOwner,
+                    name = repoName,
+                    commit = commitSha,
+                },
+            };
+
             try
             {
-                var commits = await RetryUtility.Retry(
-                    () => _client.Repository.Commit.GetAll(
-                        repoOwner,
-                        repoName,
-                        new CommitRequest { Sha = commitSha },
-                        new ApiOptions { PageCount = 1, PageSize = 100 }),
+                var response = await RetryUtility.Retry(
```

Comment:
restrict the change to this method

Reply:
If the `HttpClientUtility` can't be reused by other component, let's just remove it or keep in inside of `restore`